### DISCUSSION TITLE
How to set custom styles for the Time Picker

### DISCUSSION THREAD

**Discussion by erpoojasharmaa** [Hacker0x01/react-datepicker#3733] [Q&A]
In this version of Datepicker, https://reactdatepicker.com/#example-input-time , there seems no way to style the Time picker dropdown, like changing the font family and the background color of the time dropdown.

Is there any way to do this? Even after overriding, it has no effect.

Any help is appreciated.

Thanks.

**Comment by alejandrogonzalz** [comment]
Did you find any way to do it?
